src/Robo.php

```diff
@@ -88,14 +90,36 @@ public static function hasContainer()
     }
 
     /**
+     * Create a config object and load it from the provided paths.
+     */
+    public static function createConfiguration($paths)
+    {
+        $config = new Config();
+        static::loadConfiguration($config, $paths);
+        return $config;
+    }
+
+    /**
+     * Use a simple config loader to load configuration values from specified paths
+     */
+    public static function loadConfiguration($config, $paths)
+    {
+        $loader = new YamlConfigLoader();
+        foreach ($paths as $path) {
+            $loader->load($path);
```

Comment:
This loader doesn't have access to $config, so it can't use $config to as a reference when expanding properties in the newly loaded keys.